packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -245,13 +239,13 @@ export const prepareVaultManagerKit = (
   const topics = harden({
     asset: makePublicTopic(
       'State of the assets managed',
-      assetSubscriber,
-      storageNode,
+      assetKit.subscriber,
+      E(assetKit.recorder).getStorageNode(),
     ),
     metrics: makePublicTopic(
       'Vault Factory metrics',
-      metricsSubscriber,
-      metricsNode,
+      metricsTopicKit.subscriber,
+      E(metricsTopicKit.recorder).getStorageNode(),
```

Comment:
For now it does. I'm working towards more encapsulation because the storageNode is only the concern of the topics, not the contract. Even the child node making I want to have outside this Exo and into a "topics" maker. It'll make more sense when that's done but I don't know if I'll have time to include it in this PR. Is this a blocker?

Reply:
No, not a blocker.